cocotb/share/lib/vhpi/VhpiCbHdl.cpp

```diff
@@ -668,6 +668,11 @@ int VhpiSignalObjHdl::set_signal_value(std::string &value)
     return 0;
 }
 
+int VhpiSignalObjHdl::set_signal_value_binstr(std::string &value)
+{
+    return set_signal_value_str(value);
```

Comment:
A comment here explaining that VHPI does not distinguish these cases would be useful

Reply:
I changed the implementation so that `binstr` only works with `enum`s and `logic`s and `str` only works with `string`s.